src/main/java/io/gravitee/policy/jwt/JWTPolicy.java

```diff
@@ -166,26 +166,21 @@ public void onRequest(Request request, Response response, ExecutionContext execu
     }
 
     private String getClientId(JWTClaimsSet claims) {
-        String clientId = null;
+        Object clientIdClaim = claims.getClaim(configuration.getClientIdClaim());
```

Comment:
Do you think that if the client id claim is configured, we should also check the other (default) claims?

I suppose that if you specify a client id claim, you only want it to be tested and not the default claims.

Wdyt?